cassandra-reaper/fix.patch

```diff
@@ -0,0 +1,82 @@
+diff --git a/src/server/pom.xml b/src/server/pom.xml
```

Comment:
Can you explain these diffs so we can tell when/if they're not necessary anymore?

When there's a new upstream release of this we'll likely have to reapply these diffs and it'll be helpful to understand how and why.

Reply:
Sorry if this is me being dumb, but I'm still not sure I understand what's going on with this patch. I see that it's removing dependencies and a build step, and I'm guessing this is to get the build to exit 0.

But what effect does this have on the package? And to Jason's point, how will we know to remove this patch?

Reply:
@dlorenc can help me to explain it better why we need that

Reply:
The frontend build here includes a dependency that doesn't exist anymore on the internet, Thankfully it's just some js and css, this step disables that part.

We should check to see if this is safe to remove after the next release, the project will have to change the library here significantly.

Reply:
That makes sense, thanks.

@developer-guy can we say that in the patch file? I think the goal here is to leave good enough notes behind that the next person who comes along and looks at this knows what to do. If we can say that this might not be needed after version X, that'd help, so we don't require ourselves to dig up this PR again.